src/main/java/org/gitlab4j/api/models/Note.java

```diff
@@ -74,6 +74,9 @@ public String toString() {
     private String title;
     private String updatedAt;
     private Boolean upvote;
+    private Boolean resolved;
+    private Boolean resolvable;
+    private Participant resolvedBy;
 
```

Comment:
```type`` is also missing from the Note properties.  This should be an Enum, with the current values being "DiscussionNote" and "DiffNote"

Reply:
I cannot find the model description but I'm changing the class according to your comment!

Reply:
You'll find a sample response here:
https://docs.gitlab.com/ee/api/discussions.html#list-project-issue-discussions

Notice the items in the "notes" array have a type property.  Thanks for adding this.